test/features/config.feature

```diff
@@ -5,6 +5,5 @@ Feature: Config
         Given "Admin" as the persona
         When I log in
         And I visit "ckan-admin/config"
-        Then I should see "Data Request Config"
```

Comment:
cucumber is given when then, with and's.  Please create a then answer for this test so the logic still works